codegen/templates/deploy/crd.go

```diff
@@ -38,6 +38,9 @@ func CustomResourceDefinition(resource model.Resource) *apiextv1beta1.CustomReso
 		},
 		ObjectMeta: metav1.ObjectMeta{
 			Name: fmt.Sprintf("%s.%s", kindLowerPlural, group),
+			Annotations: map[string]string{
+				"helm.sh/hook": "crd-install",
+			},
```

Comment:
Is this a helm 2 concept? I think we should decide to support helm 3 only rather than try and be backwards compatible with a new tool

Reply:
yeah, i was unsure about this. if support is more difficult than this, i tend to agree